### PR TITLE
Update UI to match our mockup

### LINKED ISSUE #106: Implement GUI for application

### PR DESCRIPTION
**Pull request by ElginL**
Below is the updated application's UI.  The Name and Number is currently hardcoded, and can be easily changed by calling a function. It will be **updated in the future** after our create point-of-contact functionality is implemented and merged.

The transaction UI does not do anything for now as well, and will be updated in the future after our transaction functionality is implemented and merged.

image

### CODE REVIEW

**Review by notle1706 — APPROVED** [review]
LGTM however you could consider changing the font size of the POC column in the future PRs.

File: src/main/resources/view/Label.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.header-text {
+    -fx-text-fill: white;
+    -fx-font-size: 30px;
```
Comment (jicsontoh) [inline src/main/resources/view/Label.css:3]: Perhaps could consider decreasing the font size to match the other content

File: src/main/resources/view/Label.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+.description-text {
+    -fx-text-fill: white;
+    -fx-font-size: 20px;
```
Comment (jicsontoh) [inline src/main/resources/view/Label.css:8]: this could be smaller as well

**Review by jicsontoh — APPROVED** [review]
Good work, LGTM! But could consider fine tuning the font size :)

**Pull request merged**